### PR TITLE
add card without fonts

### PR DESCRIPTION
**Pull request by artem8746**

### CODE REVIEW

File: src/components/CardPhone/CardPhone.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        Apple iPhone 14 Pro 128GB Deep Purple (MQ0G3)
+      </h2>
+
+      <p className="card__price-value">$999</p>
+
+      <div className="card__params">
+        <div className="card__param-pair">
+          <p className="card__params-info">Screen</p>
+          <p className="card__params-value">6.1” OLED</p>
+        </div>
```
Comment (artem8746) [inline src/components/CardPhone/CardPhone.tsx]:
```suggestion
        <p className="card__param-pair">
          <span className="card__params-info">Screen</span>
          <span className="card__params-value">6.1” OLED</span>
        </p>
```
put <p> when it's completed thought

Comment (artem8746) [reply src/components/CardPhone/CardPhone.tsx]: change everywhere

File: src/components/CardPhone/CardPhone.tsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        <div className="card__param-pair">
+          <p className="card__params-info">RAM</p>
+          <p className="card__params-value">6 GB</p>
+        </div>
+      </div>
+
+      <div className="card__buttons">
+        <a href="#addToCart" className="card__buttons-add">
+          Add to cart
+        </a>
```
Comment (artem8746) [inline src/components/CardPhone/CardPhone.tsx]:
```suggestion
        <button href="#addToCart" className="card__buttons-add">
          Add to cart
        </button>
```

File: src/components/CardPhone/CardPhone.tsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+        <a href="#favourite" className="card__buttons-favourite">
```
Comment (artem8746) [inline src/components/CardPhone/CardPhone.tsx]:
```suggestion
        <a href="#" className="card__button-favourite">
```

File: src/components/CardPhone/cardphone.scss [hunk 4]
```diff
@@ -0,0 +1,88 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px;
+  width: 272px;
+  height: 506px;
```
Comment (artem8746) [inline src/components/CardPhone/cardphone.scss:6]:
```suggestion

```

File: src/components/CardPhone/cardphone.scss [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+
+  box-sizing: border-box;
+
+  &__image {
+    display: block;
+    margin: 0 auto;
+    margin-bottom: 8px;
+    width: 208px;
+    height: 196px;
```
Comment (artem8746) [inline src/components/CardPhone/cardphone.scss:15]:
```suggestion

```

File: src/components/CardPhone/cardphone.scss [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+    font-weight: 800px;
+    margin-bottom: 16px;
+  }
+
+  &__params {
+    display: flex;
+    flex-direction: column;
+    gap: 8px;
+    margin-bottom: 8px;
+    margin-top: 0;
```
Comment (artem8746) [inline src/components/CardPhone/cardphone.scss:35]:
```suggestion
    margin-block: 8px 0;
```

File: src/components/CardPhone/cardphone.scss [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+    color: white;
+
+    transition: background-color 0.3s ease, color 0.3s ease;
+
+    &:hover {
+      background-color: lightgrey;
+      color: black;
+    }
+  }
+}
```
Comment (artem8746) [inline src/components/CardPhone/cardphone.scss]:
```suggestion
  &__buttons {
    display: flex;
    gap: 8px;
    align-items: center;
    justify-content: space-between;

  &-add {
    display: flex;

    align-items: center;
    justify-content: center;

    width: 160px;
    height: 40px;

    text-decoration: none;
    font-weight: 700;
    font-size: 14px;
    background-color:#313237;
    color: white;

    transition: background-color 0.3s ease, color 0.3s ease;

    &:hover {
      background-color: lightgrey;
      color: black;
    }
  }
}
  }
```

**Review by sasholk — CHANGES_REQUESTED** [review]
need to rework broski🖖

**Review by l0ad1nggg — APPROVED** [review]
ok

**Pull request merged**